contents/docs/product-analytics/best-practices.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Product analytics best practices
+sidebar: Docs
+showTitle: true
+---
+
+## 1. Set up a reverse proxy
+
+Tracking and ad blockers can prevent your events from being sent to PostHog. By setting up a reverse proxy, you can send events to PostHog using your own domain. This means that events are less likely to be blocked, and so you'll be able to capture more data.
+
+See our docs on [how to set up a reverse proxy](/docs/advanced/proxy) for more details on how to do this.
+
+## 2. Implement a naming convention
+
+As your product grows, the amount of analytics data grows with it. Without naming conventions, your data quickly becomes difficult to work with.
+
+For example, let's say you're capturing signup metrics. Your team might capture events differently and with multiple names:
+
+- Web developer: Captures `Create Account` when a user clicks a button to begin signing up.
+- iOS developer: Captures the same event but names it `user_sign_up`.
+- Android developer: Calls it `create-account` but only captures it when they have finished signing up.
+
+All these differences make it more likely that you'll misinterpret your data. Implementing a naming convention prevents these problems.
+
+### Suggested naming guide
+
+- Only use lowercase letters.
+- Use present-tense verbs, e.g., "submit" and "create" instead of "submitted" and "created".
+- Use snake case, i.e., `signup_button_click`
+- Create a dedicated list of allowed verbs, and don't deviate from it. Here's a list of the most common ones you're likely to use:
+```
+- click
+- submit
+- create
+- view
+- add
+- invite
+- update
+- delete
+- remove
+- start
+- end
+- cancel
+- fail
+- generate
+- send
+```
+
+- For event names, use the `category:object_action` framework:
```

Comment:
Should we use a consistent "for example" phrasing. In this section, you use:
- e.g.
- like
- For instance
- For example

I think it would look better and more consistent if we just chose one